src/sentry/services/hybrid_cloud/user/impl.py

```diff
@@ -172,7 +174,9 @@ def get_first_superuser(self) -> Optional[RpcUser]:
             return None
         return serialize_rpc_user(user)
 
-    def get_or_create_user_by_email(self, *, email: str) -> RpcUser:
+    def get_or_create_user_by_email(
```

Comment:
@sentaur-athena Since `get_or_create_user_by_email()` is called in `getsentry`, and the function signature has changed, it might be better to use a new function name (e.g. `get_or_create_user_by_email2`) so that it's easier to make the switch on `getsentry`. 

Otherwise, the `get_or_create_user_by_email()` call will error out on `getsentry`.

Reply:
@sentaur-athena I think it'll be fine if the new arguments are optional.